### PR TITLE
Documenting the steps for promoting edges stable edges

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
From previous minor version to latest minor version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,14 @@ To see recommended feeder promotions, run:
 $ hack/stabilization-change.py
 ```
 
+##### Promoting edges from previous minor to latest minor version in stable channel
+
+To promote edges from the previous minor version to the latest minor version in the stable channel, run the following script at least 3 times when it is time and then send the pull request.
```
Comment (petr-muller) [inline README.md:85]: We could be fancy and find the transitive closure (or build this into the script entirely):

```
while ./hack/release-stable-minor.sh 4.12 && ! git diff --quiet ./channels ./internal-channels/; do
  git add ./channels/ ./internal-channels/
done
```

File: README.md [hunk 2]
```diff
@@ -80,6 +80,14 @@ To see recommended feeder promotions, run:
...
+
+```console
+$ hack/release-stable-minor.sh
```
Comment (petr-muller) [inline README.md:88]:
```suggestion
$ hack/release-stable-minor.sh <VERSION>
```

**Review by petr-muller — COMMENTED** [review]
Feel free to just accept the suggestion to add the argument to the script invocation

File: README.md (hunk 1 above)
Comment (wking) [reply README.md:85]: +1 from me to teaching the robots to do the tedious things, so we don't have to rely on docs->human to get those steps across.

**Pull request closed without merging**